data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -38,6 +44,13 @@ def __init__(self, data_source: str = None):
         self.output_score_tiles_df: pd.DataFrame
         self.output_downloadable_df: pd.DataFrame
 
+        self.yaml_fields_type_percentage_label = "percentage"
```

Comment:
🎉

Reply:
Maybe we add a note in the code linking to the issue https://github.com/usds/justice40-tool/issues/1327 ?

Reply:
Done.